web/src/pages/NewTransaction/NavigationButtons/DepositPaymentButton.tsx

```diff
@@ -55,9 +66,14 @@ const DepositPaymentButton: React.FC = () => {
     [isNativeTransaction, sendingQuantity]
   );
 
-  useEffect(() => {
-    setFinalRecipientAddress(ensResult.data || sellerAddress);
-  }, [sellerAddress, ensResult.data]);
+  const finalRecipientAddress = ensResult.data || sellerAddress;
+
+  const { data: balanceData } = useBalance({
```

Comment:
using `useBalance` for token balance in deprecated in wagmi v2, they might remove this in coming updates.
They recommend using useReadContract with token as erc20
More on this [here](https://wagmi.sh/react/guides/migrate-from-v1-to-v2#deprecated-usebalance-token-parameter)